app/src/main/java/com/realityexpander/tasky/agenda_feature/presentation/common/util/getUserAcronym.kt

```diff
@@ -0,0 +1,16 @@
+package com.realityexpander.tasky.agenda_feature.presentation.common.util
+
+fun getUserAcronym(username: String): String {
+    if(username.isBlank()) return "??"
+    if(username.length<2) return username.uppercase()
+
+    println("username: $username")
+
+    val words = username.split(" ")
+    println("words: $words, size: ${words.size}")
+    if (words.size > 1) {
+        return (words[0].substring(0, 1) + words[1].substring(0, 1)).uppercase()
+    }
```

Comment:
This logic doesn't fully seem correct. What if the user has a middle name? Then it should take the initials from the first and last name. 
Also, what if a name would accidentally have 2 spaces between first and last name, I assume the app would crash here 🤔

Reply:
AH, ok, I will update it to get the First and Last words if there are more than 1 word, not just the second word.